CMake/FindStbImage.cmake

```diff
@@ -6,13 +6,23 @@
 #  STB_IMAGE_INCLUDE_DIR - list of required include directories
 
 file(GLOB STB_IMAGE_SEARCH_PATHS
+    ${CMAKE_BINARY_DIR}/deps/
     ${CMAKE_BINARY_DIR}/deps/stb)
 find_path(STB_IMAGE_INCLUDE_DIR
-  NAMES stb_image.h stb_image_write.h
-  PATHS ${STB_IMAGE_SEARCH_PATHS})
+    NAMES stb/stb_image.h stb/stb_image_write.h
+    PATHS ${STB_IMAGE_SEARCH_PATHS})
```

Comment:
Hi, 
I was just trying to understand CMake. Just one small doubt these may be silly doubts but still.
How `stb _image_search_paths` is expanded to `/usr/include/stb/`?  Is STB_IMAGE_SEARCH_PATHS changed to  `usr/include/` automatically? 
I am not sure but I think this is not done correctly.
Please let me know where I am going wrong.

Reply:
Okay I think this is just an additional expansion to prevent errors. :+1: Sorry for the disturbance

Reply:
`${STB_IMAGE_SEARCH_PATHS}` is just a set of paths that CMake will use to guide the search. The reason I split into two blocks is just to handle when it's findable with `stb/` as a prefix on the directory name, and also when that prefix isn't there. :+1: